packages/page-parachains/src/Overview/Parachain copy.tsx

```diff
@@ -0,0 +1,175 @@
+// Copyright 2017-2022 @polkadot/app-parachains authors & contributors
```

Comment:
Why this copy?

Reply:
this can be deleted, was just using this to bring it local to see how it works.